### PR TITLE
new branch with changes collected from IR-updates

### PR DESCRIPTION
**Pull request by openprivacy**
new branch with changes collected from PR #679 - not sure if too many updates had collected for the linter to handle

### CODE REVIEW

File: docs/100-security/incident-response-checklist.md [hunk 1]
```diff
@@ -1,96 +1,130 @@
...
+   - Is a system failure causing the disruption?
 
-- Old IC: brief New IC on the situation.
-- New IC: confirm that you're taking over.
-- New IC: update the GitHub/GitLab/JIRA issue, noting that you're now the IC.
-- New IC: send out a sitrep, noting that you're taking over IC.
-- Old IC: stick around for 15-20 minutes to ensure a smooth handoff, then log off!
+2. Proceed to the next step for a confirmed incident. (For a false alarm, conclude the incident. Proceed to [_6. Conclude the incident_](#6-conclude-the-incident).)
+
+B. Assess the severity. Use the [rubric in the IR guide](incident-response-plan.md#incident-severities). (Project incidents are generally "Low severity".)
```
Comment (grugnog) [inline docs/100-security/incident-response-checklist.md:80]: What does 'Project incidents are generally "Low severity".' mean here? In my experience most project incidents are outages, and full system outages are the most common type (although rare and brief, thankfully) - would these not be high severity?

**Review by alanna-blinn — APPROVED** [review]
Much improved

File: docs/100-security/incident-response-checklist.md (hunk 1 above)
Comment (grugnog) [reply docs/100-security/incident-response-checklist.md:80]: @openprivacy @alanna-casey did you have thoughts on ^?

Comment (openprivacy) [reply docs/100-security/incident-response-checklist.md:80]: @grugnog  - You are correct that a system outage impacts the availability of services for a large number of customers, and thus is a High Severity issue. When this was written, we were primarily concerned with data and SPII breaches. We should update this.

@alanna-casey please note this for the next update which should be more minor. (Also includes the CPM cleanup and perhaps a review of the contingency plan (vs. the one in SSP-Toolkit). Thanks!

**Pull request merged**